test/markdown/heading-id.test.ts

```diff
@@ -7,7 +7,7 @@ const md = `
 # <a href="#xx" /> Foo
 ## C. Great
 ## 1. Introduction
-## ending space 
+## ending space
```

Comment:
```suggestion
## ending space 
```
This was intentional to make sure that trailing space will be ignored in id generation